.github/labels.md

```diff
@@ -0,0 +1,238 @@
+
+With the status category developers, managers or issue reporters can check if
+an issue needs further interaction and who should carry out this interaction.
+An Issue/PR usually has exactly one status.  Status may change over the lifetime
+of the Issue/PR.
+
+
+## Labels
+
+This section lists when an label should be applied and what additional actions
+may be taken.
+
+
+#### Label `category:bug`
+
+The Issue/PR describes or solves a perceived malfunction within the game.
+
+#### Label `category:feature`
+
+The Issue/PR describes or implements a new game feature.  Features are parts of
+the game which a player can interact with e.g.: A new species, a new gameplay
+mechanic or a new command line switch for the server that changes gameplay
+behavior.
+
+#### Label `category:refactoring`
+
+The Issue/PR describes or contains a new or improved implementation of an
+existing feature or any other code restructuring that doesn't modify the current
+game mechanics or representation.
+
+#### Label `category:tweak`
+
+The PR contains insignificant code changes that don't involve features,
+refactoring or bug fixing but code style grooming or value tweaking.
+
+#### Label `component:AI`
+
+The Issue/PR deals with the Python AI decision making code or affects it.
+
+#### Label `component:art music text assets`
+
+The Issue/PR deals with creating or modifying game assets or verification if the
+assets match the style goals of the game.
+
+#### Label `component:build system`
+
+The Issue/PR deals with CMake, Visual Studio, Xcode, the respective platform
+SDKs or the general build process.
+
+#### Label `component:content scripting`
+
+The Issue/PR deals with the FOCS language, turn events, the universe generator
+and probably game mechanics.
+
+#### Label `component:deployment`
+
+The Issue/PR deals with the deployment process or the software used to create
+the installer or other distribution packages.
+
+#### Label `component:documentation`
+
+The Issue/PR deals with Doxygen or primarily contains documentation.
+
+#### Label `component:game mechanic`
+
+The Issue/PR deals with the currently used or planned game mechanics.  The
+content of the PR may require extensive gameplay balance testing.
+
+#### Label `component:infrastructure`
+
+The Issue/PR deals with non-game related infrastructure that aids development.
+For example this includes issues with the project homepage, used external
+services like continuous integration or hosting of release binaries.
+
+#### Label `component:internal`
+
+The Issue/PR deals with any project component that has no explicit `component`
+label.
+
+#### Label `component:network`
+
+The Issue/PR deals with TCP/IP networking and establishing a test network
+infrastructure.
+
+#### Label `component:translation`
+
+The Issue/PR deals with the game style, game terminology, stringtable syntax
+and the translation process in general.  It maybe helpful to know the language
+of the translation to proofread.
```

Comment:
Still not absolutely sure, so again, just for clarification: As it is phrased now, I'd understand that label not only to apply to translation work in the strict sense (that is, changes/additions to non English string tables), but also to changes/additions to the default (English) in-game texts (in the default, English string table)?

Which would be an overlap with "component:art music text assets" - not that I object, just want to know if I understood that right, and if yes, if that overlap is intended.